src/main/java/io/gatling/mojo/EnterpriseStartMojo.java

```diff
@@ -48,6 +49,20 @@
 @Mojo(name = "enterpriseStart", requiresDependencyResolution = ResolutionScope.TEST)
 public final class EnterpriseStartMojo extends AbstractEnterprisePluginMojo {
 
+  public enum EnterpriseStartBehavior {
```

Comment:
```suggestion
  public static enum EnterpriseStartBehavior {
```
perhaps even a top level enum?

Reply:
Inner enums (like inner interfaces, and unlike inner classes) are always static, so the `static` keyword isn't needed.

Reply:
And code linters would complain.